kafka-client/src/main/java/dev/responsive/kafka/internal/db/ColumnName.java

```diff
@@ -30,7 +30,7 @@ public enum ColumnName {
 
   // shared partition key column
   PARTITION_KEY("partitionKey", "partitionkey"),
-  SEGMENT_ID("segmentId", "segmentid"),
+  SEGMENT_START_TIMESTAMP("segmentStartTimestamp", "segmentstarttimestamp"),
```

Comment:
@ableegoldman we're no longer running any Scylla-based soaks right? this would break compatibility

Reply:
I checked with @ableegoldman and rodesai offline, we should be OK to make this change.